src/gfx/shaders/Outline_frag.glsl

```diff
@@ -2,23 +2,50 @@ uniform sampler2D srcTex;
 uniform vec2 srcTexSize;
 varying vec2 vUv;
 
+#ifdef DEPTH_OUTLINE
+uniform sampler2D srcDepthTex; //depthTexture
```

Comment:
I think, the additional indents should be added for the code inside #ifdef construction